video-processing-service/package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "video-processing-service",
+    "version": "1.0.0",
+    "type": "module",
+    "main": "index.js",
+    "scripts": {
+        "start": "node dist/index.js",
+        "dev": "tsx watch src/index.ts",
+        "build": "tsc",
+        "lint": "eslint .",
+        "format": "prettier --write \"src/**/*.ts\"",
+        "test": "jest",
+        "test:watch": "jest --watch",
+        "test:coverage": "jest --coverage"
+    },
+    "keywords": [],
+    "author": "",
+    "license": "ISC",
+    "description": "",
+    "dependencies": {
+        "clear": "^0.1.0",
+        "express": "^4.18.2",
+        "ffmpeg": "^0.0.4"
+    },
+    "devDependencies": {
+        "@types/express": "^4.17.21",
+        "@types/jest": "^29.5.11",
+        "@types/node": "^20.10.6",
+        "@typescript-eslint/eslint-plugin": "^6.16.0",
+        "@typescript-eslint/parser": "^6.16.0",
+        "eslint": "^8.56.0",
+        "eslint-config-prettier": "^9.1.0",
+        "globals": "^13.24.0",
+        "jest": "^29.7.0",
+        "prettier": "^3.1.1",
+        "ts-jest": "^29.1.1",
+        "tsx": "^4.7.0",
+        "typescript": "^5.3.3"
```

Comment:
`nit` `non-blocking`:  notice the `^` here. This indicates that we'll take an upgraded version of this dependencies if its available. This quickly becomes problematic in the React ecosystem where many APIs have compatibility issues because of upcoming (and previous) breaking API changes. For this reason, I'll typically remove those carrots to prevent an issue happening in production.